spec/controllers/api/users_controller_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  before do 
+    @user = create(:user)
+    @token = JsonWebToken.encode(user_id: @user.id)
+  end
+  describe "GET #index" do
+    context "when valid" do 
+      before do
+        request.headers["AUTHORIZATION"] = "Bearer #{@token}"
+        get :index, format: :json
+        @json_response = JSON.parse(response.body)
+      end
+      it "returns http success" do
+        expect(response).to have_http_status(:success)
+      end
+      
+      it "JSON body response contains expected user attributes" do
```

Comment:
Keep test descriptions coherent. Here you use `it "JSON body response contains expected user attributes"` and immediately after `it "response with JSON body containing expected user email"`